packages/cadl-ranch-specs/http/azure/resource-manager/resources/mockapi.ts

```diff
@@ -61,6 +61,388 @@ const validSingletonResource = {
   },
 };
 
+const validResourceGroupExtensionsResource = {
+  id: `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}/resourceGroups/${RESOURCE_GROUP_EXPECTED}/providers/Azure.ResourceManager.Resources/topLevelTrackedResources/top/providers/Azure.ResourceManager.Resources/extensionsResources/extension`,
```

Comment:
for scope other than resource, id shouldn't contain `topLevelTrackedResources/top`